### ISSUE TITLE
Add 50 years anniversary urls to sitemap

### ISSUE THREAD

**Issue by fernandolucchesi** [equinor/energyvision#1275]

**Comment by fernandolucchesi** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Energy-Vision-5fb6386c578f1200222dd6c2/issues/equinor/energyvision/1275?planning-poker) with Zenhub @bhavi25 @padms @SvSven

**Comment by meols** [comment]
@fernandolucchesi - we had a question from TRY on when this issue can be fixed. Any input?

**Comment by fernandolucchesi** [comment]
Hi @meols, it will be fixed/released this sprint. It is not a complicated one so by next week it should be released

**Comment by meols** [comment]
@fernandolucchesi Great! Thanks :smile:

**Comment by SvSven** [comment]
@fernandolucchesi this is ready for review

**Comment by fernandolucchesi** [comment]
Hi @SvSven, please check my comments

**Comment by SvSven** [comment]
Pushed some changes based on the feedback

**Comment by fernandolucchesi** [comment]
LGTM ✅

**Comment by fernandolucchesi** [comment]
I don't think this need to be reviewed on acceptance column. lets just remove the flag and deploy
